src/coreclr/src/gc/gc.cpp

```diff
@@ -11151,9 +12070,15 @@ gc_heap::init_gc_heap (int  h_number)
 #endif //SPINLOCK_HISTORY
 
     // initialize per heap members.
+#ifndef USE_REGIONS
     ephemeral_low = (uint8_t*)1;
 
     ephemeral_high = MAX_PTR;
+#endif //!USE_REGIONS
+
+    gc_low = 0;
+
+    gc_high = 0;
```

Comment:
Nit: Is there a reason this initialization is necessary now? We set these fields in gc_heap::gc1, don't we?

Reply:
actually I don't set these in gc1 'cause the code that sets it in gc1 is only defined for regions but I still wanted to be able to assert that they are not garbage values so I init-ed them in init_gc_heap.

Reply:
I see - thanks for clarifying.